ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -137,7 +137,7 @@ def test_funnel_exclusions_full_window(self):
             "funnel_window_days": 14,
             "date_from": "2021-05-01 00:00:00",
             "date_to": "2021-05-14 00:00:00",
-            "exclusions": [{"id": "x", "type": "events", "funnel_from_step": 0, "funnel_to_step": 1},],
+            "exclusions": [{"id": "x 1 2", "type": "events", "funnel_from_step": 0, "funnel_to_step": 1},],
```

Comment:
NIT: this id is confusing because it looks like extra notation 😅

Reply:
I was just going for a space in the name, will change

Reply:
```suggestion
            "exclusions": [{"id": "x 1 name with numbers 2", "type": "events", "funnel_from_step": 0, "funnel_to_step": 1},],
```